apps/web/src/styles/globals.css

```diff
@@ -100,6 +111,36 @@ a {
   text-decoration: none;
 }
 
+h1 {
```

Comment:
All these !importants within the global CSS file are not ideal, but they were otherwise being overridden by Chakras default styles, and this happened in both the global css and in the theme.tsx, so still have them, but its only for 6 css styles.